frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -48,7 +49,7 @@ export const CsvSchemaDocumentationItem: React.FC<CsvSchemaItemProps> = ({
             Requested
           </span>
         )}
-      </h4>
+      </h3>
       <div data-testid="notes" className="margin-bottom-3">
         {item.notes?.map((note, noteIndex) => (
           <p
```

Comment:
I ran the manual Deque tests on this page, and it flagged a bunch of empty list elements - namely all the `<ul>` items for elements without an "accepted values" list. Can you also edit these, down on line 75? (GH wouldn't let me comment on that section of the file because it doesn't think it's part of the diff) 

![Screen Shot 2022-10-05 at 9 50 20 AM](https://user-images.githubusercontent.com/80282552/194116889-898d5cdb-131b-485e-b72a-471498ffb3a8.png)

Reply:
Done